torch/_torch_docs.py

```diff
@@ -166,6 +166,15 @@ def merge_dicts(*dicts):
 See :doc:`/notes/randomness` for more information.""",
 }
 
+sparse_support_notes = {
+    "sparse_beta_warning": """
+.. warning::
+    Sparse support is a beta feature and some layout(s)/dtype/device combinations may not be supported,
+    or may not have autograd support. If you notice missing functionality please
+    open an `issue<https://github.com/pytorch/pytorch/issues>`_
```

Comment:
Hm, the link still doesn't seem to render properly in the preview. I think it's also ok to say "please open a feature request" and not provide an explicit link.